### ISSUE TITLE
Homogenize commercial_partner_id string

### ISSUE THREAD

**Issue by carlosdauden** [odoo/odoo#72984]
Homogenize the string of this field so as not to cause confusion when it is used in the filters

Diferent string "Partner Company" (ok in help...):

https://github.com/odoo/odoo/blob/7e81134c712cc659e6b006d4b6de63725aa5bc1f/addons/account/report/account_invoice_report.py#L23

Same string "Commercial Entity":

https://github.com/odoo/odoo/blob/fabc59ff05ab300d69bc6f2b05b851d485698af9/odoo/addons/base/models/res_partner.py#L220-L221

https://github.com/odoo/odoo/blob/4e1e8d46ac1c8bf1fac17562119171c79179cc19/addons/account/models/account_move.py#L139-L140

**Comment by pedrobaeza** [comment]
I don't think this can be done in stable, but @qdp-odoo it would be good to have it on master branch.
